src/schemas/execute.yaml

```diff
@@ -0,0 +1,121 @@
+BatchCallArgs:
+  title: Arguments for batch call
+  type: object
+  required:
+    - block
+    - calls
+  properties:
+    block:
+      title: Block tag
+      $ref: '#/components/schemas/BlockNumberOrTag'
+    stateOverrides:
+      title: State overrides
+      $ref: '#/components/schemas/StateOverrides'
```

Comment:
This feels out of place here.  What is the reasoning for having this as a parameter?  It also *greatly* complicates implementation in some clients I believe compared to not including it so I would like to see a compelling reason for its inclusion.

Reply:
This is a greatly used feature of eth_call. I agree it is less necessary in multicall since you can build up the state in previous calls. However not all state mutations are possible (e.g. simply replacing a contract code).

I agree this is extra work for client devs. However I would like to hear from them if this is indeed a *complication*. It seems to me all have a way to build a temporary state for serving eth_call. This goes a step further to modify parts of that state.

Reply:
I think the state override feature is probably useful enough to warrant the complexity. I can imagine a security researcher breaking a problem down into two parts. If this value ever can become X, there is a vulnerability to report. They could research that aspect first (as it might be simpler) then research "can this value ever become X".

Could also fake signers of a multi-sig that use store approvals (as many do).